src/bp/ui-studio/src/web/components/Shared/Utils/ElementPreview.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { fetchContentItem } from '~/actions'
+import store from '~/store'
+
+import withLanguage from '../../Util/withLanguage'
+
+const ElementPreview = props => {
+  const { itemId, contentItem, contentLang } = props
+  if (!itemId) {
+    return null
+  }
+
+  if (!contentItem) {
+    props.fetchContentItem(props.itemId.replace('#!', ''))
+  }
+
+  return contentItem ? `${contentItem.schema.title} | ${contentItem.previews[contentLang]}` : ''
+}
+
+const mapStateToProps = ({ content: { itemsById } }, { itemId }) => ({ contentItem: itemsById[itemId] })
+const ConnectedElementPreview = connect(
```

Comment:
have you tried passing store using mapStateToProps ?

Reply:
Nop, but i'm not sure it would work, maybe by playing around it could be possible. Other connected components are made the same way so i'll leave them this way for now